test/regression.uts

```diff
@@ -2453,7 +2503,6 @@ l = [ p for p in RawPcapReader(f) ]
 assert len(l) == 1
 
 = Check RawPcapReader on pcap
-~ pcap
```

Comment:
Can you explain why you are removing multiple `~ pcap` lines?

Reply:
While testing, noticed that some pcap tests didn't have the `~ pcap` keyword, so I added the keyword manually for some of them. Then discovered that keyword could be simply applied to the whole test set instead, so added this:
```
############
############
+ pcap / pcapng format support
~ pcap
```
Finally when squashing the commits, I though it would be ok to simply delete the `~ pcap` keyword from individual tests, given that it was covered already by the test set.

Reply:
I think that you should keep the old `pcap` tag the way there are.

Reply:
ok, reverted the existing ones. the new pcap tag can stay under test-set or shall be changed/removed?

Reply:
You can keep the new tag. Thanks.